app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -48,9 +48,11 @@ def self.selected_remote(idv_session:)
       if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
          IdentityConfig.store.in_person_proofing_enabled &&
          idv_session.service_provider&.in_person_proofing_enabled
-        idv_session.skip_doc_auth == false
+        idv_session.skip_doc_auth_from_how_to_verify == false ||
+          idv_session.skip_doc_auth == false
       else
-        idv_session.skip_doc_auth.nil? ||
+        idv_session.skip_doc_auth_from_how_to_verify.nil? ||
+          idv_session.skip_doc_auth_from_how_to_verify == false || idv_session.skip_doc_auth.nil? ||
           idv_session.skip_doc_auth == false
```

Comment:
I wonder if safe navigation or a dig could be used here to eliminate the `.nil?` checks. If that is the case wouldn't really need to the if else here right? because both blocks would be the same.

Reply:
I did try the with `dig` but it didn't work 😦  Also, we do have a follow up ticket to revisit this soon so we'll have another chance to refactor this. Thanks!

Reply:
Okay that sounds good. We can revisit this later.